packages/excalidraw/components/App.tsx

```diff
@@ -4105,11 +4109,12 @@ class App extends React.Component<AppProps, AppState> {
       this.setState({ isBindingEnabled: true });
     }
     if (isArrowKey(event.key)) {
-      const selectedElements = this.scene.getSelectedElements(this.state);
-      const elementsMap = this.scene.getNonDeletedElementsMap();
-      isBindingEnabled(this.state)
-        ? bindOrUnbindSelectedElements(selectedElements, this)
-        : unbindLinearElements(selectedElements, elementsMap);
+      bindOrUnbindLinearElements(
```

Comment:
@dwelle @mtolmacs We will need to do something about the `keyUp` handler, as it triggers after each `keyDown` in our colour picker, command palette and now font picker, blocking the UI thread.

#8057